scripts/toggle.sh

```diff
@@ -49,22 +53,29 @@ cmd=("${@:$OPTIND}")
 
 opened="$(showvariable @__popup_opened)"
 
-if [[ -n "$opened" && ("$opened" = "$name" || -z "$*") ]]; then
+if [[ -n "$opened" ]]; then
 	# Clear the variables to prevent a manually attached session from being
 	# detached by the keybinding.
 	tmux set -u @__popup_opened \; detach
-else
-	name="${name:-"$DEFAULT_NAME"}"
-	socket_name="$(get_socket_name)"
-	id_format="$(showopt @popup-id-format "$DEFAULT_ID_FORMAT")"
-	popup_id="$(format @popup_name "$name" "$id_format")"
-
-	eval "tmux -C \; $(showhook @popup-before-open) >/dev/null"
-	tmux popup "${popup_args[@]}" "
+
+	exit 0
+fi
+
+name="${name:-"$DEFAULT_NAME"}"
+socket_name="${socket_name:-$(get_socket_name)}"
+id_format="${id_format:-$(showopt @popup-id-format "$DEFAULT_ID_FORMAT")}"
+on_init="${on_init:-$(showopt @popup-on-init "$DEFAULT_ON_INIT")}"
```

Comment:
For hooks, we should use `showhook`, which performs some necessary processing, such as handling line breaks.